token/class_test.go

```diff
@@ -0,0 +1,130 @@
+	'7':  Decimal | NonZeroDecimal | Octadecimal | Hexadecimal | Ident,
+	'8':  Decimal | NonZeroDecimal | Hexadecimal | Ident,
+	'9':  Decimal | NonZeroDecimal | Hexadecimal | Ident,
+	'A':  Letter | Uppercase | Hexadecimal | IdentStart | Ident,
+	'B':  Letter | Uppercase | Hexadecimal | IdentStart | Ident,
+	'C':  Letter | Uppercase | Hexadecimal | IdentStart | Ident,
+	'D':  Letter | Uppercase | Hexadecimal | IdentStart | Ident,
+	'E':  Letter | Uppercase | Hexadecimal | IdentStart | Ident,
+	'F':  Letter | Uppercase | Hexadecimal | IdentStart | Ident,
+	'G':  Letter | Uppercase | IdentStart | Ident,
+	'H':  Letter | Uppercase | IdentStart | Ident,
+	'I':  Letter | Uppercase | IdentStart | Ident,
+	'J':  Letter | Uppercase | IdentStart | Ident,
+	'K':  Letter | Uppercase | IdentStart | Ident,
+	'L':  Letter | Uppercase | IdentStart | Ident,
+	'M':  Letter | Uppercase | IdentStart | Ident,
+	'N':  Letter | Uppercase | IdentStart | Ident,
+	'O':  Letter | Uppercase | IdentStart | Ident,
+	'P':  Letter | Uppercase | IdentStart | Ident,
+	'Q':  Letter | Uppercase | IdentStart | Ident,
+	'R':  Letter | Uppercase | IdentStart | Ident,
+	'S':  Letter | Uppercase | IdentStart | Ident,
+	'T':  Letter | Uppercase | IdentStart | Ident,
+	'U':  Letter | Uppercase | IdentStart | Ident,
+	'V':  Letter | Uppercase | IdentStart | Ident,
+	'W':  Letter | Uppercase | IdentStart | Ident,
+	'X':  Letter | Uppercase | IdentStart | Ident,
+	'Y':  Letter | Uppercase | IdentStart | Ident,
+	'Z':  Letter | Uppercase | IdentStart | Ident,
+	'_':  IdentStart | Ident,
+	'a':  Letter | Lowercase | Hexadecimal | IdentStart | Ident,
+	'b':  Letter | Lowercase | Hexadecimal | IdentStart | Ident,
+	'c':  Letter | Lowercase | Hexadecimal | IdentStart | Ident,
+	'd':  Letter | Lowercase | Hexadecimal | IdentStart | Ident,
+	'e':  Letter | Lowercase | Hexadecimal | IdentStart | Ident,
+	'f':  Letter | Lowercase | Hexadecimal | IdentStart | Ident,
+	'g':  Letter | Lowercase | IdentStart | Ident,
+	'h':  Letter | Lowercase | IdentStart | Ident,
+	'i':  Letter | Lowercase | IdentStart | Ident,
+	'j':  Letter | Lowercase | IdentStart | Ident,
+	'k':  Letter | Lowercase | IdentStart | Ident,
+	'l':  Letter | Lowercase | IdentStart | Ident,
+	'm':  Letter | Lowercase | IdentStart | Ident,
+	'n':  Letter | Lowercase | IdentStart | Ident,
+	'o':  Letter | Lowercase | IdentStart | Ident,
+	'p':  Letter | Lowercase | IdentStart | Ident,
+	'q':  Letter | Lowercase | IdentStart | Ident,
+	'r':  Letter | Lowercase | IdentStart | Ident,
+	's':  Letter | Lowercase | IdentStart | Ident,
+	't':  Letter | Lowercase | IdentStart | Ident,
+	'u':  Letter | Lowercase | IdentStart | Ident,
+	'v':  Letter | Lowercase | IdentStart | Ident,
+	'w':  Letter | Lowercase | IdentStart | Ident,
+	'x':  Letter | Lowercase | IdentStart | Ident,
+	'y':  Letter | Lowercase | IdentStart | Ident,
+	'z':  Letter | Lowercase | IdentStart | Ident,
+}
+
+func TestClass(t *testing.T) {
```

Comment:
__[gometalinter]__
* [gocyclo] cyclomatic complexity 12 of function TestClass() is high (> 10)

<!-- Comment by [SideCI](https://sideci.com) -->